flatland/envs/rail_env.py

```diff
@@ -774,3 +773,51 @@ def close(self):
             except Exception as e:
                 print("Could Not close window due to:", e)
             self.renderer = None
+
```

Comment:
see a few lines above:
`
def close(self):
        """
        This method closes any renderer window.
`